js/parts-more/PackedBubbleSeries.js

```diff
@@ -422,87 +1086,243 @@ seriesType(
+                valMin = Math.min(valMin, Math.min.apply(this, s.yData));
+            });
 
-        // Calculate radius of bubbles in series.
+            zMin = zMin || valMin;
+            zMax = zMax || valMax;
+            return [zMin, zMax];
+        },
+        /**
+         * Calculate radius of bubbles in series.
+         */
         getPointRadius: function () { // bubbles array
 
             var series = this,
                 chart = series.chart,
                 plotWidth = chart.plotWidth,
                 plotHeight = chart.plotHeight,
                 seriesOptions = series.options,
+                useSimulation = seriesOptions.useSimulation,
                 smallestSize = Math.min(plotWidth, plotHeight),
                 extremes = {},
                 radii = [],
                 allDataPoints = chart.allDataPoints,
                 minSize,
                 maxSize,
                 value,
-                radius;
-
+                radius, zExtremes;
             ['minSize', 'maxSize'].forEach(function (prop) {
                 var length = parseInt(seriesOptions[prop], 10),
-                    isPercent = /%$/.test(length);
+                    isPercent = /%$/.test(seriesOptions[prop]);
 
                 extremes[prop] = isPercent ?
                     smallestSize * length / 100 :
-                    length;
+                    length * Math.sqrt(allDataPoints.length);
             });
 
-            chart.minRadius = minSize = extremes.minSize;
-            chart.maxRadius = maxSize = extremes.maxSize;
+            chart.minRadius = minSize = extremes.minSize /
+                Math.sqrt(allDataPoints.length);
+            chart.maxRadius = maxSize = extremes.maxSize /
+                Math.sqrt(allDataPoints.length);
+
+            zExtremes = useSimulation ?
+                this.calculateZExtremes() :
+                [minSize, maxSize];
 
             (allDataPoints || []).forEach(function (point, i) {
 
-                value = point[2];
+                value = useSimulation ?
+                    Math.max(Math.min(point[2], zExtremes[1]), zExtremes[0]) :
+                    point[2];
 
                 radius = series.getRadius(
-                    minSize,
-                    maxSize,
+                    zExtremes[0],
+                    zExtremes[1],
                     minSize,
                     maxSize,
                     value
                 );
-
-                if (value === 0) {
+                if (radius === 0) {
                     radius = null;
                 }
-
                 allDataPoints[i][2] = radius;
                 radii.push(radius);
             });
-
             this.radii = radii;
         },
+        // Draggable mode:
+        /**
+         * Redraw halo on mousemove during the drag&drop action.
+         *
+         * @param {Highcharts.Point} point The point that should show halo.
+         */
+        redrawHalo: function (point) {
```

Comment:
Looks similar to `networkgraph.redrawHalo` - synergy here? :)

Reply:
Added as Mixin :)